pkg/apic/service.go

```diff
@@ -114,21 +117,41 @@ func (c *ServiceClient) UpdateSubscriptionDefinitionPropertiesForCatalogItem(cat
 	return c.updateSubscriptionDefinitionPropertiesForCatalogItem(catalogItemID, propertyKey, subscriptionSchema)
 }
 
-// postApiServiceUpdate - called after APIService was created or updated.
-// Update description and title after updating or creating APIService to include the stage name if it exists
-func (c *ServiceClient) postAPIServiceUpdate(serviceBody *ServiceBody) {
-	if serviceBody.Stage != "" {
-		addDescription := fmt.Sprintf("StageName: %s", serviceBody.Stage)
-		if len(serviceBody.Description) > 0 {
-			serviceBody.Description = fmt.Sprintf("%s, %s", serviceBody.Description, addDescription)
-		} else {
-			serviceBody.Description = addDescription
+// updateAPIServiceRevisionTitle - called after APIService gets created or updated.
+// Update title after creating or updating APIService according to the APIServiceRevision Pattern
+func (c *ServiceClient) updateAPIServiceRevisionTitle(serviceBody *ServiceBody) {
+	title := c.cfg.GetAPIServiceRevisionPattern() // "{{APIServiceName}} - {{date:YYYY/MM/DD}} - r {{version}}"
+	version := strconv.Itoa(serviceBody.serviceContext.revisionCount + 1)
```

Comment:
Not quite sure if this is correct, revision and version are different

Reply:
I took the variable `version` directly from the story {{ version }}. I spoke with Cyrille and have changed the wording to {{revision}}.